package.json

```diff
@@ -10,21 +10,23 @@
   "license": "MIT",
   "dependencies": {
     "@faker-js/faker": "6.2.0",
-    "fictional": "0.4.5"
+    "fictional": "0.4.5",
+    "uuid": "8.3.2"
```

Comment:
Seeing as this is a lib, should we use ranges rather than pinning so that people are able to get the later versions of transitive deps? Otherwise we could be the blocker for security upgrades, for e.g.

I opted for pinning since its what we've done so far, but thought would be good to discuss.

Reply:
I think this comes down to trust, and whilst we can assume good actors, I think we should consider bad actors, and I would rather have a human make the decision here than semver. I know we lock down the VM that runs inside Snaplet, but I would just want to be overly cautious.

Reply:
But, I think we should discuss this in realtime.